py-deploy/tldeploy/identity.py

```diff
@@ -340,6 +348,22 @@ def _meta_transaction_function_call(self, signed_meta_transaction: MetaTransacti
             signed_meta_transaction.signature,
         )
 
+    def get_meta_transaction_status(self, identity_address, hash):
+        identity_contract = self._get_identity_contract(identity_address)
+
+        # the filter cannot handle bytes32 values as hex strings, use HexBytes()
+        meta_tx_execution_logs = identity_contract.events.TransactionExecution.getLogs(
+            fromBlock=0, argument_filters={"hash": HexBytes(hash)}
```

Comment:
Can we expose this as default argument please: `, *, from_block=0, to_block="latest"` (or whatever the default is)